src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.md

```diff
@@ -64,7 +64,7 @@ Configured to show a table:
     rgbIndex={1}
     name='Table Demo'
     sqlOperator='='
-    sql='SELECT * FROM syn11346063.14'
+    sql='SELECT assay, study, dataType, tissue, consortium FROM syn11346063.14'
```

Comment:
Changed example to show that the programmatic options sql changes this to SELECT * FROM syn11346063.14 ...